### DISCUSSION TITLE
Arduino Nano Connect compile error

### DISCUSSION THREAD

**Discussion by dallaby23** [JAndrassy/ArduinoOTA#195] [Q&A] [answered]
hope this is something simple.. I have a Nano Connect board flashed with Pico Core and when I use the Wifi101_OTA example I get the following error.. Also I have copied the platform.local.txt to C:\Users\username\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.1.0 

I apologize if I missed something simple here.

C:\Users\Password Auction\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.1.0\libraries\ArduinoOTA\src\ArduinoOTA.cpp:30:10: fatal error: include/UdpContext.h: No such file or directory
   30 | #include "include/UdpContext.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Multiple libraries were found for "ArduinoOTA.h"
  Used: C:\Users\Password Auction\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.1.0\libraries\ArduinoOTA
  Not used: C:\Users\Password Auction\Documents\Arduino Projects\libraries\ArduinoOTA
exit status 1

Compilation error: exit status 1

**Comment by JAndrassy** [comment]
delete the bundled ArduinoOTA library. it is is only for Pico W

**Reply by dallaby23** [reply]
I deleted that but same error when compiling

C:\Users\Password Auction\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.1.0\libraries\ArduinoOTA\src\ArduinoOTA.cpp:30:10: fatal error: include/UdpContext.h: No such file or directory
   30 | #include "include/UdpContext.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

exit status 1

Compilation error: exit status 1

**Reply by dallaby23** [reply]
I tried adding #include <WiFiUdp.h> then got a ton of other errors referencing wifininna so I tried removing that library and still can't get it to compile..

**Comment by JAndrassy** [comment, marked as answer]
you didn't delete it
C:\Users\Password Auction\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\3.1.0\libraries\ArduinoOTA\src\ArduinoOTA.cpp

**Reply by dallaby23** [reply]
sorry I misunderstood which one to delete.. I removed the one suggested and reinstalled the ArduinoOTA library and Wifinina .. it does compile now (thank-you for that) but does not work. Seems to hang when calling WiFi.localIP() ... I will have to look into that separately.. thanks for getting me this far though.
